lib/c.c

```diff
@@ -529,11 +529,16 @@ int fputc(int c, FILE *stream)
 /* Non-portable: Assume page size is 4KiB */
 #define PAGESIZE 4096
 
+#define CHUNK_SIZE_FREED_BIT_MASK 1
```

Comment:
Simply name the identifiers as mask, which represents bit mask. That is, `CHUNK_SIZE_FREED_MASK`.